app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java

```diff
@@ -31,5 +31,4 @@ public String getStatus() {
         return result;
     }
 
-
```

Comment:
revert the whole file

Reply:
Okay.

Reply:
Done.

Reply:
It's not yet done. This file should not be included in the pr